2024-02-15 13:56:35,408 INFO    MainThread:1072499 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_setup.py:_flush():76] Configure stats pid to 1072499
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_setup.py:_flush():76] Loading settings from /home/cvlab08/.config/wandb/settings
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_setup.py:_flush():76] Loading settings from /home/cvlab08/projects/jinlovespho/github/monodepth/MaskingDepth/wandb/settings
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program_abspath': '/home/cvlab08/projects/jinlovespho/github/monodepth/MaskingDepth/train.py', 'program': '/home/cvlab08/projects/jinlovespho/github/monodepth/MaskingDepth/train.py'}
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_init.py:_log_setup():526] Logging user logs to /home/cvlab08/projects/jinlovespho/github/monodepth/MaskingDepth/wandb/run-20240215_135635-b7wh4py4/logs/debug.log
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_init.py:_log_setup():527] Logging internal logs to /home/cvlab08/projects/jinlovespho/github/monodepth/MaskingDepth/wandb/run-20240215_135635-b7wh4py4/logs/debug-internal.log
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_init.py:init():566] calling init triggers
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'Train': {'model_name': 'pho_model', 'log_path': './log/20240126', 'use_cuda': True, 'seed': 41, 'start_epoch': 0, 'end_epoch': 50, 'batch_size': 8, 'num_workers': 1, 'lr': '1e-4', 'd_consistency': 0.0, 'f_consistency': 0.0, 'K': 1, 'wandb': True, 'load_optim': False, 'model': {'baseline': 'DPT', 'max_depth': 80.0, 'load_weight': False, 'weight_path': './weight', 'mlp_head': False}, 'add_model': {'uncert': False, 'load_weight': False, 'weight_path': './weight', 'threshold': 0.7}, 'data': {'height': 192, 'width': 640, 'dataset': 'kitti_depth', 'num_prev_frame': 0, 'splits': 'eigen_temp', 'data_path': '/home/cvlab08/projects/data/KITTI', 'img_ext': '.jpg'}}}
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_init.py:init():616] starting backend
2024-02-15 13:56:35,409 INFO    MainThread:1072499 [wandb_init.py:init():620] setting up manager
2024-02-15 13:56:35,410 INFO    MainThread:1072499 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-15 13:56:35,411 INFO    MainThread:1072499 [wandb_init.py:init():628] backend started and connected
2024-02-15 13:56:35,414 INFO    MainThread:1072499 [wandb_init.py:init():720] updated telemetry
2024-02-15 13:56:35,492 INFO    MainThread:1072499 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-15 13:56:36,078 INFO    MainThread:1072499 [wandb_run.py:_on_init():2254] communicating current version
2024-02-15 13:56:36,140 INFO    MainThread:1072499 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.16.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-02-15 13:56:36,140 INFO    MainThread:1072499 [wandb_init.py:init():804] starting run threads in backend
2024-02-15 13:56:38,063 INFO    MainThread:1072499 [wandb_run.py:_console_start():2233] atexit reg
2024-02-15 13:56:38,064 INFO    MainThread:1072499 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-02-15 13:56:38,064 INFO    MainThread:1072499 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-02-15 13:56:38,064 INFO    MainThread:1072499 [wandb_run.py:_redirect():2178] Redirects installed.
2024-02-15 13:56:38,064 INFO    MainThread:1072499 [wandb_init.py:init():847] run started, returning control to user process
2024-02-15 13:56:48,563 WARNING MsgRouterThr:1072499 [router.py:message_loop():77] message_loop has been closed
